ironmon_tracker/MoveData.lua

```diff
@@ -1,3756 +1,3768 @@
+		power = Constants.NO_POWER,
 		pp = "10",
 		accuracy = "90",
-		category = MoveCategories.STATUS,
+		category = MoveData.Categories.STATUS,
 		summary = "Lowers the target's Speed stat by two stages.",
 	},
 	{
 		id = "185",
 		name = "Faint Attack", -- This is "Feint Attack" in gen 6+
-		type = PokemonTypes.DARK,
+		type = PokemonData.Types.DARK,
 		power = "60",
 		pp = "20",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.SPECIAL,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.SPECIAL,
 		summary = "Deals damage and bypasses accuracy checks to always hit, unless the target is in the semi-invulnerable turn of a move such as Dig or Fly.",
 	},
 	{
 		id = "186",
 		name = "Sweet Kiss",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "10",
 		accuracy = "75",
-		category = MoveCategories.STATUS,
+		category = MoveData.Categories.STATUS,
 		summary = "Causes the target to become confused for 2-5 turns. 50% chance to damage self as a 40 power Physical move.",
 	},
 	{
 		id = "187",
 		name = "Belly Drum",
-		type = PokemonTypes.NORMAL,
-		power = NOPOWER,
+		type = PokemonData.Types.NORMAL,
+		power = Constants.NO_POWER,
 		pp = "10",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		summary = "The user loses half of their max HP, and in return raises their Attack stat all the way to +6 stages.",
 	},
 	{
 		id = "188",
 		name = "Sludge Bomb",
-		type = PokemonTypes.POISON,
+		type = PokemonData.Types.POISON,
 		power = "90",
 		pp = "10",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		summary = "Deals damage and has a 30% chance of poisoning the target.",
 	},
 	{
 		id = "189",
 		name = "Mud-Slap",
-		type = PokemonTypes.GROUND,
+		type = PokemonData.Types.GROUND,
 		power = "20",
 		pp = "10",
 		accuracy = "100",
-		category = MoveCategories.PHYSICAL,
+		category = MoveData.Categories.PHYSICAL,
 		summary = "Deals damage and lowers the target's accuracy by one stage.",
 	},
 	{
 		id = "190",
 		name = "Octazooka",
-		type = PokemonTypes.WATER,
+		type = PokemonData.Types.WATER,
 		power = "65",
 		pp = "10",
 		accuracy = "85",
-		category = MoveCategories.SPECIAL,
+		category = MoveData.Categories.SPECIAL,
 		summary = "Deals damage and has a 50% chance to lower the target's accuracy by one stage.",
 	},
 	{
 		id = "191",
 		name = "Spikes",
-		type = PokemonTypes.GROUND,
-		power = NOPOWER,
+		type = PokemonData.Types.GROUND,
+		power = Constants.NO_POWER,
 		pp = "20",
-		accuracy = PLACEHOLDER,
-		category = MoveCategories.STATUS,
+		accuracy = Constants.BLANKLINE,
+		category = MoveData.Categories.STATUS,
 		summary = "Creates a hazard for the enemy team (can stack 3 times). Switching out deals 1/8, 1/6, or 1/4 max HP in damage to Pokemon without Flying or Levitate.",
```

Comment:
I feel like the wording here could be mis-interpreted as "the pokemon that is switching out takes damage" rather than "pokemon take damage when switched in"